### PR TITLE
[4.0] com_banners sidebar

### PR DESCRIPTION
**Pull request by brianteeman**
removes unused sidebar code from com_banners

### CODE REVIEW

**Comment by brianteeman** [comment]
Updated branch to retrigger drone time out errors unrelated to the code in this pr

File: administrator/components/com_banners/tmpl/banners/default.php [hunk 1]
```diff
@@ -34,12 +34,7 @@
 ?>
 <form action="<?php echo Route::_('index.php?option=com_banners&view=banners'); ?>" method="post" name="adminForm" id="adminForm">
 	<div class="row">
-		<?php if (!empty($this->sidebar)) : ?>
-            <div id="j-sidebar-container" class="col-md-2">
-				<?php echo $this->sidebar; ?>
-            </div>
-		<?php endif; ?>
-        <div class="<?php if (!empty($this->sidebar)) {echo 'col-md-10'; } else { echo 'col-md-12'; } ?>">
+        <div class="col-md-12">
```
Comment (Quy) [inline administrator/components/com_banners/tmpl/banners/default.php]:
```suggestion
		<div class="col-md-12">
```

File: administrator/components/com_banners/tmpl/clients/default.php [hunk 2]
```diff
@@ -33,12 +33,7 @@
 ?>
 <form action="<?php echo Route::_('index.php?option=com_banners&view=clients'); ?>" method="post" name="adminForm" id="adminForm">
 	<div class="row">
-		<?php if (!empty($this->sidebar)) : ?>
-            <div id="j-sidebar-container" class="col-md-2">
-				<?php echo $this->sidebar; ?>
-            </div>
-		<?php endif; ?>
-        <div class="<?php if (!empty($this->sidebar)) {echo 'col-md-10'; } else { echo 'col-md-12'; } ?>">
+        <div class="col-md-12">
```
Comment (Quy) [inline administrator/components/com_banners/tmpl/clients/default.php]:
```suggestion
		<div class="col-md-12">
```

File: administrator/components/com_banners/tmpl/tracks/default.php [hunk 3]
```diff
@@ -19,12 +19,7 @@
 ?>
 <form action="<?php echo Route::_('index.php?option=com_banners&view=tracks'); ?>" method="post" name="adminForm" id="adminForm">
 	<div class="row">
-		<?php if (!empty($this->sidebar)) : ?>
-            <div id="j-sidebar-container" class="col-md-2">
-				<?php echo $this->sidebar; ?>
-            </div>
-		<?php endif; ?>
-        <div class="<?php if (!empty($this->sidebar)) {echo 'col-md-10'; } else { echo 'col-md-12'; } ?>">
+        <div class="col-md-12">
```
Comment (Quy) [inline administrator/components/com_banners/tmpl/tracks/default.php]:
```suggestion
		<div class="col-md-12">
```

**Comment by Quy** [comment]
Delete?
https://github.com/joomla/joomla-cms/blob/65f4725ec5e059afa1d14d97ad80aab6c822c98a/administrator/components/com_banners/View/Tracks/HtmlView.php#L120-L121

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 1d5f1deca63218535251b507c3051262124632a4<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24869">issues.joomla.org/tracker/joomla-cms/24869</a>.</sub>

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on 1d5f1deca63218535251b507c3051262124632a4<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24869">issues.joomla.org/tracker/joomla-cms/24869</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24869">issues.joomla.org/tracker/joomla-cms/24869</a>.</sub>

**Comment by roland-d** [comment]
Thank you

**Comment by brianteeman** [comment]
Thanks @roland-d

**Pull request merged**